endpoints/openrtb2/auction.go

```diff
@@ -1823,6 +1845,30 @@ func validateDevice(device *openrtb2.Device) error {
 	return nil
 }
 
+func validateExactlyOneInventoryType(reqWrapper *openrtb_ext.RequestWrapper) error {
```

Comment:
This function is well covered by integration tests, but can we add a `TestValidateExactlyOneInventoryType` anyways?

Reply:
@guscarreon Pushed requested test below